tithemi/assignment3/Test.cpp

```diff
@@ -0,0 +1,70 @@
+#include "gtest/gtest.h"
+#include "Dictionary.hpp"
+#include "LettersGrid.hpp"
+
+static const std::vector<std::string> words =
+{
+    "CAR", "CARD", "CART", "CAT"
+};
+
+static const std::vector<std::string> prefixes =

```

Comment:
You're only using prefixes in one test case, consider making this a local variable of that test case.